src/main/resources/db/changelog/changes/2-create-info-role-table-changelog.sql

```diff
@@ -0,0 +1,14 @@
+--liquibase formatted sql
+
+--changeset noyan:1
+
+CREATE TABLE `roles` (
+     `id` bigint NOT NULL AUTO_INCREMENT,
+     `create_date` datetime(6) DEFAULT NULL,
```

Comment:
please change datetime types to timestamp.

Reply:
I just changed datetime to timestamp